src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/DueDiligenceDto.java

```diff
@@ -114,4 +116,16 @@ public String getDiligence() {
     public void setDiligence(String diligence) {
         this.diligence = diligence;
     }
+
+    public boolean isEmpty() {
+        return Objects.isNull(identityDate) &&
+               StringUtils.isBlank(name) &&
+               Objects.isNull(address) &&
```

Comment:
If we do go down this route, this line would/should probably call an isEmpty() method on the address object, which would need implementing.

Reply:
A task for another ticket eprhaps.

Reply:
Maybe. What does the web send down here, out of interest, when the due diligence block is empty/unpopulated? A null or an address block with all empty fields?

Reply:
10 new unit tests

Reply:
Thanks for adding those tests.

What does the web send though please, when block is unpopulated? And could you name the specific test(s) that check this scenario?

Reply:
In the web we have:     
// Empty OverseasEntityDueDiligence object
    setApplicationData(req.session, {}, OverseasEntityDueDiligenceKey);

Same for the other block. 

The tests in DueDiligenceDataBlockValidatorTest deal with the different combinations and permutations of this scenario.

Reply:
Where you see "new DueDiligenceDto()" in the argument list that is an empty object.